crates/bevy_gltf/src/loader.rs

```diff
@@ -116,6 +120,11 @@ fn load_buffers(buffers: iter::Buffers, asset_path: &Path) -> Result<Vec<Vec<u8>
         match buffer.source() {
             Source::Uri(uri) => {
                 if uri.starts_with("data:") {
+                    if uri.starts_with("data:application/octet-stream;base64,") {
+                        buffer_data.push(base64::decode(&uri[37..])?);
```

Comment:
lets move "data:application/octet-stream;base64," to a `const octet_stream_uri: &str` and then use `octet_stream_uri.len()` instead of `37`, otherwise its just a "magic number".

str::len() is const so theres no perf overhead.

Reply:
Good point, it should be better now.